backend/models/post.py

```diff
@@ -347,6 +348,20 @@ def remove_subscriber(self, user):
         if user.key in self.subscribers and self.author != user.key:
             self.subscribers.remove(user.key)
 
+    def create_notification_message(self, user_key, current_institution_key, sender_institution_key=None):
+        """ Create message that will be used in notification.
+            user_key -- The user key that made the action.
+            current_institution -- The institution that user was in the moment that made the action.
+            sender_institution_key -- The institution by which the post was created,
+                if didn't define the sender institution  should be the current institution. 
```

Comment:
if it hasn't been defined yet, the sender institution should be the current institution..